toolkit/templates/user-research-consent-banner.html

```diff
@@ -0,0 +1,11 @@
+<div class="user-research-banner">
+  <div class="wrapper">
+    <p>
+      <strong class="banner-heading">Help us improve the Digital Marketplace</strong>
+      <a href="#" class="user-research-banner-close-btn">Close</a>
+    </p>
+    <p>
+      <a href="{{ url_for('external.user_research_consent') }}">Sign up to be a potential user research participants</a>
```

Comment:
Is the `s` on the end of the sentence correct?

Reply:
Good catch